app/main.py

```diff
@@ -1,52 +1,68 @@
-import json
-import xml.etree.ElementTree as ET
-
-
-class Book:
-    def __init__(self, title: str, content: str):
-        self.title = title
-        self.content = content
-
-    def display(self, display_type: str) -> None:
-        if display_type == "console":
-            print(self.content)
-        elif display_type == "reverse":
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown display type: {display_type}")
-
-    def print_book(self, print_type: str) -> None:
-        if print_type == "console":
-            print(f"Printing the book: {self.title}...")
-            print(self.content)
-        elif print_type == "reverse":
-            print(f"Printing the book in reverse: {self.title}...")
-            print(self.content[::-1])
-        else:
-            raise ValueError(f"Unknown print type: {print_type}")
-
-    def serialize(self, serialize_type: str) -> str:
-        if serialize_type == "json":
-            return json.dumps({"title": self.title, "content": self.content})
-        elif serialize_type == "xml":
-            root = ET.Element("book")
-            title = ET.SubElement(root, "title")
-            title.text = self.title
-            content = ET.SubElement(root, "content")
-            content.text = self.content
-            return ET.tostring(root, encoding="unicode")
-        else:
-            raise ValueError(f"Unknown serialize type: {serialize_type}")
+from app.book_service.book import Book
+from app.book_service.display_types import (
+    DisplayProcessor,
+    ConsoleDisplayProcessor,
+    ReverseDisplayProcessor
+
+)
+from app.book_service.print_book_types import (
+    PrintProcessor,
+    ConsolePrintProcessor,
+    ReversePrintProcessor
+)
+from app.book_service.serializers import (
+    SerializeProcessor,
+    JSONSerializeProcessor,
+    XMLSerializeProcessor
+)
+
+
+def create_display_processor(display_type: str) -> DisplayProcessor:
+    if display_type == "console":
+        return ConsoleDisplayProcessor()
+    elif display_type == "reverse":
+        return ReverseDisplayProcessor()
+    else:
+        raise ValueError(f"Unknown display type: {display_type}")
+
+
+def create_print_processor(print_type: str) -> PrintProcessor:
+    if print_type == "console":
+        return ConsolePrintProcessor()
```

Comment:
All you displayers, printers, serializers have static methods, then why do you return class instance?